src/ucx_plugin.c

```diff
@@ -210,14 +210,43 @@ static ncclResult_t GetSocketAddr(union ncclSocketAddress *addr) {
   return ncclSuccess;
 }
 
+static ncclResult_t ucx_config_no_cuda(ucp_config_t *config) {
+  char tls[PATH_MAX] = "^cuda";
+  const char *ucx_tls;
+  ssize_t n;
+
+  ucx_tls = getenv("NCCL_UCX_TLS");
+  if (ucx_tls == NULL) {
+    ucx_tls = getenv("UCX_TLS");
+  }
+
+  if (ucx_tls != NULL) {
+    n = snprintf(tls, sizeof(tls), "^cuda,%s", ucx_tls);
```

Comment:
afair this will discard all transports listed, including user's config

Reply:
now checking if specification is positive, negative or default